api/src/pcapi/core/mails/transactional/sendinblue_template_ids.py

```diff
@@ -39,7 +39,7 @@ class TransactionalEmail(Enum):
         id_prod=253, id_not_prod=134, tags=["changement_email_confirmation"], use_priority_queue=True
     )
     EMAIL_CHANGE_CANCELLATION = models.Template(
-        id_prod=1001, id_not_prod=136, tags=["changement-email_annulation_suspension_compte"], use_priority_queue=True
+        id_prod=1001, id_not_prod=136, tags=["changement_email_annulation_suspension_compte"], use_priority_queue=True
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Warning là dessus !
Cette valeur est utilisée par le marketing directement sur Brevo
Il faut voir avec eux (Katy) si c'est ok de changer ça set que ça ne casse rien dans leurs stats

Même si je suis d'accord avec ta proposition de naming

Reply:
ok j'attend ça réponse avant de rajouter un commit pour ça